src/main/java/com/teragrep/pth_06/planner/walker/conditions/LatestCondition.java

```diff
@@ -93,4 +94,9 @@ public boolean equals(final Object object) {
         final LatestCondition cast = (LatestCondition) object;
         return this.value.equals(cast.value);
     }
+
+    @Override
+    public int hashCode() {
+        return Objects.hashCode(value);
```

Comment:
wrong method used Objects.hashCode(value) while should use Objects.hash(value)

Reply:
changed to Objects.hash(value)